.eslintrc.js

```diff
@@ -37,22 +37,13 @@ module.exports = {
     'no-unused-vars': 'off',
     'prefer-object-spread': 'off',
     'spaced-comment': 'off',
-    'prettier/prettier': [
-      'error',
-      {
-        arrowParens: 'always',
-        printWidth: 100,
-        singleQuote: true,
-        trailingComma: 'none',
-        plugins: ['prettier-plugin-package']
```

Comment:
I see that `prettier-plugin-package` has been removed everywhere, but it hasn't been reimplemented anywhere. this is a hard block for me.

Reply:
see the other comment. I'll check it's actually being loaded, and if not will bring this back where needed